### ISSUE TITLE
Is this still under active development?

### ISSUE THREAD

**Issue by feutl** [RainLoop/rainloop-webmail#1736]
I just have no other way to go, therefor using github seems the only way.

Is Rainloop still under active development. I like the simplicity of Rainloop and how it "interacts" with Imap and Contacts and basically no need for a DB (except contacts).
I want to switch my family members to a own Rainloop installation soon, but look at the repository I have mixed feelings

* 615 open Issues
* 13 pull request
Some of this dates back to 2014

But still
* 484 forks
* 2523 stars and
* 177 watcher which seems not that bad.

Can someone give some insights?
Thanks

**Comment by billyprice1** [comment]
I assume yes it was on 1.5.18 so i assume yes and the last github update was 2 days ago

**Comment by feutl** [comment]
If I look through the git repo it just doesn't seem that way, the plugins are from years ago - for example the recaptcha plugin was modified a year ago, no code is that stable I assume.

Nevertheless, you are right, code is being changed, the latest version is from May, this is legit.

Still, what about all the open issues nobody addresses since ages, they could have been closed, or tagged or placed somewhere in a timeline. What about the push requests, they could have been declined at least.

I see lots of tickets got closed >800, but somehow I just felt the need to ask that question, and you should always trust your gut ;)

Thanks

**Comment by billyprice1** [comment]
I agree with you but this is quite a big project its a community project think about bootstrap as a base that project is a live https://github.com/twbs/bootstrap yet look how many issues it have it's due to the size of the project

**Comment by feutl** [comment]
Thanks, but the comparison didn't make it better ;)
Just looked at the github insights ;) and bootstrap sees more code changes and updates in a week than rainloop has seen in the last year. And still, bootstrap open issues around 350, rainloop around 600.

Also the only one communicating to me is you, which I appreciate, but no one of the "active" development team jumps in?
It feel strange, but I hope it is worth it, using rainloop and getting part of the community over the time.

**Comment by DeepDiver1975** [comment]
Open source is about doing it. Fork and submit pull requests. This is the way to get involved.
My 2 cents.

**Comment by feutl** [comment]
You are right, but not everyone is a coder.

But no worries, I try my best to get involved, and this is the start. Installing and using a product, setting things up, and giving back something - and even if it is just this discussion.

**Comment by megadr01d** [comment]
In the beginning the dev [asked us more](https://github.com/RainLoop/rainloop-webmail/issues?q=is%3Aissue+author%3ARainLoop+is%3Aclosed) to help shape Rainloop  Anyway, [not abandoned](https://github.com/RainLoop/rainloop-webmail/issues/1640). Slow but under development.  Dev has a specific views and guidelines for Rainloop, but he accepts fixes PRs, more than feature requests.

However in the open source world, an application [doesn't really belong](https://github.com/RainLoop/rainloop-webmail/issues/1736#issuecomment-405157305) to anyone. 

Why such mixed feelings? Live the moment. Rainloop installation is pretty easy and straight forward. At its present state it's pretty stable. In terms of features, don't keep any expectations. What you have now is probably what you'll have in the future. It works (with a quirk or two, as anything else) and it's way faster and slicker than Roudcube. Has also great keybinding support, out of the box. You might also be interested in Cypht.

**Comment by feutl** [comment]
I reopen it, because it is, once again a valid question ;) and the impression evolves that this does not seem to get a lot of love.
Lots of issues open, no PRs approved

@RainLoop could you just bring some light into that darkness?

**Comment by billyprice1** [comment]
The last edit was 3 months ago for the .docker file as i said once before having a huge project is it's due to the size of the project and lack of people supporting it. and what needs changing really

**Comment by feutl** [comment]
I look at those changes as well, but this is not something which makes the project active developed.
There are 744 open issues and 8 PRs which should be somehow addressed, even if they just get dismissed with a statement why.

And what is missing - hmm - most people realize this if it is too late I would say, but this huge project is for sure not without errors, mistakes and security issues.

Would be nice to get some feedback by @RainLoop somehow somewhere, there is not much activity by @RainLoop itself.

**Comment by jaggeri** [comment]
I asked the developer a question on support@rainloop.net weeks ago and there was no response. I wont buy it unless I can get some setup support. Which makes me think, is Rainloop really open source - or just one guy? 

Maybe it can it be released to Mozilla and the Thunderbird team? I am trying to replace Thunderbird with a private Web Based email solution like Gmail, or Roundcube. Rainloop seems simpler to set up, and less resource demands, but was hoping to set it up on a Win 10 box.

Any suggestions welcome

**Comment by feutl** [comment]
You could go with https://afterlogic.com/ which seems to be the same than rainloop. As far as I understood there is/was some relation between that company and the rainloop developer.
Anyhow, they offer a solution which is under active development.

For me, this is not a option, because I use rainloop within my nextcloud using a [plugin from pierre allain](https://github.com/pierre-alain-b/rainloop-nextcloud) and Afterlogic is not willing to make their implementation simple usable for nextcloud. For me, I would not mind to pay something for software that just works and gets developed. But this seems not the truth for rainloop.

**Comment by winkelement** [comment]
@RainLoop please give at least SOMEONE write access so that person can review and merge the pull requests which will then hopefully lead to more PR's.

**Comment by jaggeri** [comment]
Thanks for the link feutl. I had a look at afterlogic, but I am just small potatoes compared to their prestigious client list. I was hoping to pay a couple hundred $ for  a small ( 5 users) windows 10 web mail server setp up support on a modest Win 10 box. Ongoing online support for bugs, back ups or the occasional glitch would be comforting.

The future seems clear, I dont know why Mozilla are still developing TB email apps that are trapped in a desktop. I guess I will need to wait another 5 years and see what happens. Meanwhile there is always Gmail and their AI spybot.

**Comment by feutl** [comment]
@jaggeri 

They do have a free / lite version as well which is open source
https://github.com/afterlogic/webmail-lite-8

**Comment by jaggeri** [comment]
oh - I didnt see that.
I will have a look
Thanks!
